GroupManager/Utils/ClaimPrincipalExtension.cs

```diff
@@ -0,0 +1,174 @@
+﻿using Microsoft.Identity.Client;
```

Comment:
Don't we want to have a multi-platform Microsoft.Identity.Client.Web ?

Reply:
Not sure what you mean here Jean-Marc, please explain a bit more. Thanks

---
In reply to: [271643283](https://github.com/Azure-Samples/active-directory-dotnet-admin-restricted-scopes-v2/pull/16#discussion_r271643283) [](ancestors = 271643283)

Reply:
I'm saying that a lot of the code you have in the Utils subfolder is the same as the Microsoft.Identity.Client.Web DLL. If we provided that library as multi-platform (.NET core and .NET Framework), this would avoid copy/paste;

Reply:
I tried, building a multi-platform library is a task in itself :) Could we start that as a separate task please?

---
In reply to: [272098656](https://github.com/Azure-Samples/active-directory-dotnet-admin-restricted-scopes-v2/pull/16#discussion_r272098656) [](ancestors = 272098656)